langserve/server.py

```diff
@@ -681,10 +681,23 @@ async def _stream() -> AsyncIterator[dict]:
                     ) from validation_exception
 
             try:
+                config_w_callbacks = config.copy()
+                event_aggregator = AsyncEventAggregatorCallback()
+                config_w_callbacks["callbacks"] = [event_aggregator]
+                has_sent_metadata = False
                 async for chunk in runnable.astream(
                     input_,
-                    config=config,
+                    config=config_w_callbacks,
                 ):
+                    if not has_sent_metadata and event_aggregator.callback_events:
+                        yield {
+                            "event": "metadata",
+                            "data": {
```

Comment:
Do we not need to serialize the data into json? I think it doesn't do this by default for streaming

Reply:
ah yeah that's a good callout - that makes sense why the output had single quotes instead of double quotes! I'll make this change before pushing

Reply:
Before: 
<img width="934" alt="Screenshot 2023-11-06 at 9 52 08 PM" src="https://github.com/langchain-ai/langserve/assets/9028897/adbd2d73-06f9-42ea-8be9-b00f456b403c">

After:
<img width="939" alt="Screenshot 2023-11-06 at 9 51 49 PM" src="https://github.com/langchain-ai/langserve/assets/9028897/d6a04bfa-7c9a-4af8-a881-c84c4f41d0f0">


Looks like both were able to be interpreted as JSON, but maybe there are issues on other payloads / it looks a lot better in my postman console now